tests/WalletAPI.test.ts

```diff
@@ -310,4 +310,30 @@ describe ('Test of Stoa API for the wallet with `sample_data`', () => {
 
         assert.deepStrictEqual(expected, response.data);
     });
+
+    it ('Test of the path /wallet/transactions/history - Filtering - exclude DataPayload in specific filter', async () =>
```

Comment:
There was one when it was `payload`, but nothing is returned when it is `outbound`.